css/style.css

```diff
@@ -0,0 +1,717 @@
+@import url('https://fonts.googleapis.com/css2?family=Heebo&family=PT+Sans&family=Roboto:wght@500&family=Rowdies:wght@300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Heebo&family=PT+Sans&display=swap');
+body{
+    margin:0 auto;
+    font-family: 'Heebo', sans-serif;
+    font-family: 'PT Sans', sans-serif;
+    font-family: 'Roboto', sans-serif;
+    font-family: 'Rowdies', cursive;
+
+}
+
+.headermain{
+    height: 4rem;
+    display: flex;
+    flex-direction: row;
+    box-shadow: 0 8px 6px -6px lightblue;background-color: lightblue;
+}
+
+.headertitle{
+    padding:0.5rem;
+    font-size: 2.5rem;
+    
+}
+
+.headernav{
+    list-style-type: none;
+    margin-left:auto;
+    display: flex;
+    flex-direction: row;
+  
+}
+.headernav-items{
+    display: inline;
+    font-size: 1.5rem;
+    padding-right:0.5rem;
+}
+.headernav-items-img{
+    width: 100%;
+    height: 100%;
+    object-fit: contain;
+}
+.footer-container{
+
+    height:auto;
+background-color: black;
+margin-bottom: 0;
+color: white;
+font-family: 'Heebo', sans-serif;
+font-family: 'PT Sans', sans-serif;
```

Comment:
Take care of indentation
Google style guide suggests indentation of 2 spaces for HTML and css

Reply:
Taken care, thanks for comments